muffnn/mlp/tests/test_mlp_classifier.py

```diff
@@ -261,3 +263,20 @@ def test_refitting():
     y_binary = (y == y[0]).astype(float)
     clf.fit(X, y_binary)
     assert np.array_equal(clf.classes_, np.unique(y_binary))
+
+
+def test_cross_val_predict():
+    # Make sure it works in cross_val_predict for multiclass.
+
+    X, y = load_iris(return_X_y=True)
+    y = LabelBinarizer().fit_transform(y)
+    X = StandardScaler().fit_transform(X)
+
+    mlp = MLPClassifier(n_epochs=10,
+                        solver_kwargs={'learning_rate': 0.05},
+                        random_state=4567).fit(X, y)
+
+    y_oos = cross_val_predict(mlp, X, y, cv=4, method='predict_proba')
+    auc = roc_auc_score(y, y_oos, average=None)
+
+    assert np.all(auc >= 0.96)
```

Comment:
I do care. This is more of an integration test. I forgot to seed the CV iterator. Once everything is seeded, the test won't be brittle.